src/CMakeLists.txt

```diff
@@ -61,7 +61,7 @@ foreach(test ${gtest_targets})
 endforeach()
 
 if(TARGET UNIT_gz_TEST)
-  # Running `gz gazebo` on macOS has problems when run with /usr/bin/ruby
+  # Running `gz gui` on macOS has problems when run with /usr/bin/ruby
```

Comment:
Should this be `gz sim` since the original was `gz gazebo`?

Reply:
Good catch. I think the change was made in https://github.com/gazebosim/gz-gui/pull/538. We'll have to fix it there as well.

Reply:
Done in https://github.com/gazebosim/gz-gui/pull/545/commits/3efe68417897daaeff7617b9b700583d34a3fda8